lib/internal/test_runner/coverage.js

```diff
@@ -477,7 +477,7 @@ function sortCoverageFiles(a, b) {
 
 function setupCoverage(options) {
   let originalCoverageDirectory = process.env.NODE_V8_COVERAGE;
-  const cwd = process.cwd();
+  const cwd = options.cwd ?? process.cwd();
```

Comment:
Should there be an internal assertion that `cwd` is a string here?

Reply:
Hey @RedYetiDev, currently, the only validation we have for this option is here: https://github.com/pmarchini/node/blob/feat/cwd-option-run/lib/internal/test_runner/runner.js#L565.

We can add validation everywhere it's received, but are we sure that it's needed?